components/header/podIndicator/styles.js

```diff
@@ -31,11 +31,17 @@ const styles = (theme) =>
       fontWeight: theme.typography.fontWeightMedium,
     },
     indicatorPrompt: {
+      display: "inline-flex",
       color: theme.palette.text.secondary,
       textTransform: "none",
       fontSize: "0.825rem",
       fontWeight: theme.typography.fontWeightRegular,
     },
+    indicatorName: {
+      textOverflow: "ellipsis",
+      overflow: "hidden",
+      maxWidth: "200px",
```

Comment:
I wasn't sure on the convention for units to use here — can switch to theme spacing/rem/etc.

Reply:
I'd use rem; `10rem` should do it.

You'll also want a `white-space: nowrap` to prevent names with spaces creating a second line of text.

![Screen Shot 2020-10-23 at 8 57 04 AM](https://user-images.githubusercontent.com/175515/97006678-3520ce80-150e-11eb-9b51-c8b7b668422b.png)

![Screen Shot 2020-10-23 at 8 58 02 AM](https://user-images.githubusercontent.com/175515/97006684-36ea9200-150e-11eb-9c6a-4a991ef74814.png)

Lastly, this needs a bit of restructuring - because it's in the same `div` as the logo, long names push the icons out. The pod location indicator should be in a separate div, after everything else, so the content can flow above it:

![Screen Shot 2020-10-23 at 9 00 07 AM](https://user-images.githubusercontent.com/175515/97006661-305c1a80-150e-11eb-9e8f-c348193f98eb.png)

Reply:
yep, just noticed the wrapping issue (had a change in dev tools that wasn't committed 🤦‍♂️) Cool, will add that restructuring too 👍

Reply:
actually, if the name _isn't_ supposed to be in the same div as the logo, does it need truncating at all (unless it's more than full width of the screen?)

Reply:
Only so the carat is obvious. Maybe truncate 50% of the screen?

Reply:
(With a min-width, too.)

Reply:
The nav menu options are supposed to be full height, with icons vertically centred, they look odd shunted up to the top to accommodate a full width pod name(Files is focussed to show clickable area):

![Screenshot 2020-10-23 at 14 39 04](https://user-images.githubusercontent.com/8266322/97010833-d6734900-153d-11eb-9b4a-be21b3ec9509.png)

Can obviously override or update prism styles, but not sure how much I should be fighting against it…

Reply:
The PodIndicator is still part of the same div as logo, so it won't render the name underneath the icons.

I'm not 100% sure whether having the name go below the icons is the design we want though, maybe double-check with @KyraAssaad ?

Reply:
Got confirmation from @KyraAssaad on slack to go with truncation (rather than flowing underneath and disrupting alignment) _for now_, until she gets a chance to have a better think about how the nav layout works. Also confirmed the max-width chosen seems reasonable, and we're not too worried about that pushing the nav _slightly_